guillotina/api/service.py

```diff
@@ -36,6 +42,31 @@ def __setattr__(self, name, value):
 
 
 class Service(View):
+    __validator__ = None
+
+    async def validate(self):
+        data = await self.request.json()
+        if(self.__validator__ is None):
+            try:
+                requestBody = self.__config__['requestBody']
+                self.__schema__ = requestBody['content']['application/json']['schema']['$ref'].split('/')[-1]
+                self.__validator__ = get_schema_validator(self.__schema__)
+            except Exception as e:
+                logger.warning('Invalid jsonschema: {}'.format(e))
```

Comment:
If there is an error, how about we set `self.__validator__` to a sentinel value like `_sentinel = object()` so we won't continue to output the warning + retry calculating schema?

I can show you this if this doesn't make sense.

Reply:
Do you mean a flag kind of thing? `if self.__validator__ = "invalidschema"` then if wont give warning or retry calculating schema?

Reply:
Close. Take a look at how aiohttp does it.

- https://github.com/aio-libs/aiohttp/blob/master/aiohttp/helpers.py#L81
- https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client.py#L196